### DISCUSSION TITLE
Importing records "#tags Description" vs "Description #tags" how to ?

### DISCUSSION THREAD

**Discussion by Derf000** [almarklein/timetagger#297] [Q&A] [answered]
Just a question  about importing data : 
how to get  display like in demo/sandbox app :  Descriptions #tag1 #tag2 (Description first, tags follows)

When importing records data
I always get tags first and description after (#tag1 #tag2 Descriptions strings )
I have try to change order in "columns" when importing : 
<key;start;stop;tags;description> vs <key;start;stop;description;tags>
but nothing help.

Also when exporting records I note that tags get repeated in the description fields (in front or at end) : 
example from exported demo records : 
key	start	stop	tags	description
wZPtUavf	2022-01-03 08:20:00	2022-01-03 10:27:00	#admin #client2	Did some stuff #client2 #admin
xKWoBJRi	2022-01-03 10:37:00	2022-01-03 11:50:00	#client2 #code #writing	Did some things #client2 #code #writing
bZKqcTiZ	2022-01-03 12:00:00	2022-01-03 13:38:00	#admin #client2	Did some work #client2 #admin
Why tags are repeated in the description field ? 

any comment / response are welcome :-)

**Comment by almarklein** [comment]
When importing a record that has a description and tags defined in different columns, the tags are put in the description at the start. If you don't want this, you can put the tags into description yourself, and don't use the tag-column in the import.

> Why tags are repeated in the description field ? 

This is because the tags function a bit like what's called "project" in other tools. By also putting the tags in a seperate column, one can e.g. convert the tags to a project name before processing things further.

**Comment by Derf000** [comment, marked as answer]
Thanks for this explanation 👍

**Comment by Tagirijus** [comment]
Is there any way to import tags as secondary tags?

**Reply by almarklein** [reply]
No, you have to mark them as secondary afterwards.

**Reply by Tagirijus** [reply]
Ok, what a bummer. Yet thanks for your answer!
